### DISCUSSION TITLE
ScriptReference/Cursor.SetCursor

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#554] [Notes]
# ScriptReference/Cursor.SetCursor

https://docs.unity3d.com/ScriptReference/Cursor.SetCursor.html

**Comment by vertxxyz** [comment]
Hotspot coordinates are pixels. Your texture should be imported as a Cursor in the import settings.
If you are using UI Toolkit you can use the [`cursor`](https://docs.unity3d.com/Manual/UIE-USS-SupportedProperties.html#cursor) property in USS to flexibly override the cursor based on your elements and UI states.

**Comment by Consalv0** [comment]
On windows the size of the cursor is capped by the system cursor size.
You can get this info using the Registry if you want to change to a higher resolution.

```
Microsoft.Win32.RegistryKey registryKey = Microsoft.Win32.Registry.CurrentUser.OpenSubKey("Control Panel\\Cursors");
var cursorSize = registryKey.GetValue( "CursorBaseSize" );
```
